docs/notary/setup-Komodo-Notary-Node.md

```diff
@@ -371,6 +371,8 @@ bitcoind &
 ### Start Komodo and all the Smart Chains
 
 ```bash
+cp -f ~/dPoW/iguana/assetchains.old ~/komodo/src/
+chmod +x ~/komodo/src/assetchains.old
```

Comment:
hmm, if there is no execution rights for ~/dPoW/iguana/assetchains.old atm probably it's better to fix it in dPoW repo

Reply:
ok, let me do that

Reply:
pushed the change to this PR https://github.com/KomodoPlatform/dPoW/pull/150